### PR TITLE
Fixed the message to not appear when no data is loaded

### LINKED ISSUE #7102: Annoying message on File > Close Data Book
This message appears even if no data was loaded, and when you click on Close Data Book. See screenshot below
image

And there is a typo error in the first line of the message, it should be Are you sure you want to close `your data` rather than `you data`

### PR DESCRIPTION
**Pull request by N-thony**
Fixes #7102 
@dannyparsons this PR fixed the small bug related to the corresponding issue. Please take a look. Thanks.
@africanmathsinitiative/developers this is ready for review. Thanks.

### CODE REVIEW

File: instat/frmMain.vb [hunk 1]
```diff
@@ -1578,14 +1578,13 @@ Public Class frmMain
 
     Private Sub mnuFileCloseData_Click(sender As Object, e As EventArgs) Handles mnuFileCloseData.Click
         If Not bDataSaved Then
-            If DialogResult.No = MsgBox("Are you sure you want to close you data?" &
+            If ucrDataViewer.GetWorkSheetCount() = 0 OrElse DialogResult.Yes = MsgBox("Are you sure you want to close your data?" &
                                          Environment.NewLine & "Any unsaved changes will be lost.",
                                          MessageBoxButtons.YesNo, "Close Data") Then
```
Comment (Patowhiz) [inline instat/frmMain.vb:1583]: @N-thony would be ideal to pass this `"Are you sure you want to close your data?"` and `"Any unsaved changes will be lost."` through the translations function. 
You don't necessarily have to do it in this PR and I'm also not sure if we the 2 corresponding translations in the database. (@lloyddewit  could easily add the translations later and not have to change this line)

Comment (N-thony) [reply instat/frmMain.vb:1583]: @Patowhiz we don't have the 2 corresponding translations in the database that is the reason I made directly the change here.

Comment (dannyparsons) [reply instat/frmMain.vb:1583]: Good point, we should work on having all our messages going through the translation system. Can be done separately from this PR.

**Review by dannyparsons — DISMISSED** [review]
Looks good to me

File: instat/frmMain.vb [hunk 2]
```diff
@@ -1578,7 +1578,7 @@ Public Class frmMain
 
     Private Sub mnuFileCloseData_Click(sender As Object, e As EventArgs) Handles mnuFileCloseData.Click
         If Not bDataSaved Then
-            If ucrDataViewer.GetWorkSheetCount() = 0 OrElse DialogResult.Yes = MsgBox("Are you sure you want to close your data?" &
+            If ucrDataViewer.GetWorkSheetCount() = 0 OrElse DialogResult.Yes = MsgBox("Are you sure you want to close you data?" &
```
Comment (shadrackkibet) [inline instat/frmMain.vb:1581]: Looks like you reverted the typo correction?

**Review by shadrackkibet — CHANGES_REQUESTED** [review]

File: instat/frmMain.vb (hunk 2 above)
Comment (N-thony) [reply instat/frmMain.vb:1581]: Yes, as @Patowhiz, I will fix that in another PR. But first I will check with @lloyddewit for the possibility.

Comment (shadrackkibet) [reply instat/frmMain.vb:1581]: Okay

**Review by shadrackkibet — DISMISSED** [review]

File: instat/frmMain.vb (hunk 1 above)
Comment (lloyddewit) [reply instat/frmMain.vb:1583]: `MsgBox` text is not currently translated. Some `MsgBox` text is multiline and/or contains variable values. This text will be more difficult to translate.
I suggest that we implement the `MsgBox` translation mechanism, across all of R-Instat, in a separate PR.
In summary, I think @N-thony was correct to fix this typo here.

File: instat/frmMain.vb (hunk 2 above)
Comment (lloyddewit) [reply instat/frmMain.vb:1581]: @N-thony your original typo fix was correct (see my comment above)
```suggestion
            If ucrDataViewer.GetWorkSheetCount() = 0 OrElse DialogResult.Yes = MsgBox("Are you sure you want to close your data?" &
```

**Review by lloyddewit — APPROVED** [review]

**Comment by lloyddewit** [comment]
@shadrackkibet @rdstern Please could one of you test/merge? thanks

**Review by shadrackkibet — APPROVED** [review]

**Pull request merged**